sdk/monitor/azure-monitor-opentelemetry-exporter/tests/trace/test_trace.py

```diff
@@ -418,6 +419,23 @@ def test_span_to_envelope_client_http(self):
         envelope = exporter._span_to_envelope(span)
         self.assertEqual(envelope.data.base_data.target, "www.wikipedia.org")
 
+        # Stable semconv
```

Comment:
So, does this test involve both conventions now? Shouldn't we separate scenarios?

Reply:
Or at least add new ones.

Reply:
Already multiple scenarios in one super test. Make todo